### ISSUE TITLE
Master checklist?

### ISSUE THREAD

**Issue by daveliepmann** [clojurebridge-berlin/organization#161]
One of the major questions I had as a new member of the orga-team was "what...do we need to do?" The big picture is daunting. One solution I think could help is a master checklist. It would be shaped like this:

> 
> ### Catering
>  - [ ] Installfest food
>     - [ ] get bids from vendors
>     - [ ] compare bids
>  - [ ] Workshop food
>     - [ ] get bids from vendors
>     - [ ] compare bids
> 
> ### Venue (etc.)
>  - [ ] task
>  - [ ] ...

Previously the team tried Trello, and now we're using Projects, but I wonder if a single document with an overview of all tasks could be useful. One major benefit that a single checklist would have over our other project management solutions is that it would make obvious to all contributors which tasks are not done that they could help with. It might help with keeping everyone up-to-date about status.

How do we feel about this approach compared to other project management tools?

**Comment by plexus** [comment]
This could be a start: https://github.com/clojurebridge-berlin/organization/wiki/Tasks-and-roles

**Comment by vsmart** [comment]
+1, I totally think this would be a great tool for everyone --- esp seeing what's on the roadmap this week, next month, etc. 

Re " it would make obvious to all contributors which tasks are not done that they could help with"  --- I wonder how the checklist would make that more clear than the project board. Do you think it might be helpful to have an `up-for-grab` or `help-needed` tag on the GH issues? I specifically created a bunch of issues far in advance for this reason, but maybe people need more encouragement and more visual help to see which issues are open for help and need it.

**Comment by daveliepmann** [comment]
In my opinion the obstacle to obviousness is the abstraction layer of GH issues. To get from "project overview" to "status of individual items" one must click into each individual issue. A checklist makes the overview and the individual items the same view.

**Comment by vsmart** [comment]
Hm, but the status of the item is reflected by its position in the board, right? (i.e. "done", "in progress", "up next" etc)

**Comment by daveliepmann** [comment]
Yes, but that status is abstracted over multiple individual tasks, which is
lossy

**Comment by vsmart** [comment]
👍 Yea, I definitely agree that an overall checklist provides that quick one-look overview that the project board does not necessarily, and it also gets rid of this "Are we forgetting anything important?" feeling that at least I feel occasionally. :)

**Comment by daveliepmann** [comment]
Having a single master document would help for lookups, too. For instance, right now I'm trying to look up the workshop venue address. Which ticket is it in? Is the ticket closed? Why is this kind of basic info spread across multiple pages?
